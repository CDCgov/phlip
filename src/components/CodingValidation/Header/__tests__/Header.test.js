import React from 'react'
import { shallow } from 'enzyme'
import { Header } from '../index'

const props = {
  projectName: 'Test Project',
  projectId: 1,
  currentJurisdiction: {
    startDate: '2018-01-01T10:00:00',
    endDate: '2018-01-18T05:00:00'
  }
}

describe('Coding scene --- Header component', () => {
  test('should render correctly', () => {
    expect(shallow(<Header {...props} />)).toMatchSnapshot()
  })
})
