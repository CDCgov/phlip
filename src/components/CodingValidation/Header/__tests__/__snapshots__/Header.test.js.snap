// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Coding scene --- Header component should render correctly 1`] = `
<Container
  alignItems="center"
  column={false}
  flex={false}
  spacing={0}
  style={
    Object {
      "padding": "20px 27px",
    }
  }
>
  <Column
    component={
      <WithStyles
        item={true}
      />
    }
    displayFlex={false}
    flex={false}
    style={
      Object {
        "display": "flex",
        "paddingRight": 5,
      }
    }
  >
    <IconButton
      aria-label="Go back"
      color="black"
      iconSize={28}
      onClick={[Function]}
      tooltipText=""
    >
      arrow_back
    </IconButton>
  </Column>
  <WithStyles
    color="inherit"
    type="title"
  >
    <span
      style={
        Object {
          "paddingRight": 10,
        }
      }
    >
      Coding
    </span>
    <span
      style={
        Object {
          "color": "#0faee6",
        }
      }
    >
      Test Project
    </span>
  </WithStyles>
  <React.Fragment>
    <span />
    <div
      style={
        Object {
          "flex": "1",
        }
      }
    >
      <WithStyles />
    </div>
    <Column
      component={
        <WithStyles
          item={true}
        />
      }
      displayFlex={false}
      flex={true}
    >
      <WithStyles
        align="right"
        color="default"
        type="caption"
      >
        Segment start 
        <span
          style={
            Object {
              "color": "black",
            }
          }
        >
          2018-1-1
        </span>
      </WithStyles>
      <WithStyles
        align="right"
        color="default"
        type="caption"
      >
        Segment end 
        <span
          style={
            Object {
              "color": "black",
            }
          }
        >
          2018-1-18
        </span>
      </WithStyles>
    </Column>
    <Column
      component={
        <WithStyles
          item={true}
        />
      }
      displayFlex={false}
      flex={true}
    />
    <WithTheme
      to="/project/1/protocol"
    >
      <WithTheme
        aria-label="View and edit protocol"
        style={
          Object {
            "backgroundColor": "white",
            "color": "black",
          }
        }
        value="View/Edit Protocol"
      />
    </WithTheme>
  </React.Fragment>
</Container>
`;
